demos/notebooks/demo_pipeline.ipynb

```diff
@@ -610,10 +1023,736 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Clean up open resources\n",
+    "We have a few resources we have left open we should take care of.\n",
+    "\n",
+    "## Shut down cluster \n",
+    "To free up processing resources, let's shut down the cluster in case it is still open. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cm.stop_server(dview=multiprocessing_pool)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Shut down logger, optionally remove log files\n",
+    "If you set up your logger to log to files, and you don't want to preserve them, you can delete them with the followingIf you customized your log filenames in Appendix 1 you may have to change the `log_files` pattern for the following to work. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shut down logger (otherwise will not be able to delete it)\n",
+    "logging.shutdown()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "delete_logs = True\n",
+    "logging_dir = cm.paths.get_tempdir() \n",
+    "if delete_logs:\n",
+    "    log_files = glob.glob(logging_dir + '\\\\demo_pipeline' + '*' + '.log')\n",
+    "    for log_file in log_files:\n",
+    "        print(f\"Deleting {log_file}\")\n",
+    "        os.remove(log_file)\n",
+    "else:\n",
+    "    print(f\"If you want to inspect your logs they are in {logging_dir}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Appendices\n",
+    "A few things that would have interrupted the flow of the main notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=logging_explained></a>\n",
+    "## Appendix 1: Logging\n",
+    "Python has a powerful built-in [logging module](https://docs.python.org/3/library/logging.html) for generating log messages while a program is running. It lets you generate custom log messages, and set a threshold to determine which logs you will see. You will only receive messages above the severity threshold you set: you can choose from: `logging.DEBUG`, `logging.INFO`, `logging.WARNING`, `logging.ERROR`, or `logging.CRITICAL`. For instance, setting the threshold to `logging.DEBUG` will print out every logging statement, while setting it to `logging.ERROR` will print out only errors and critical messages. This system gives much more flexibility and control than interspersing `print()` statements throughought your code when debugging. \n",
```

Comment:
tighten phrasing if we need this paragraph at all

Reply:
Moved to docs